tutorials/profiler/generate.py

```diff
@@ -0,0 +1,339 @@
+import pytest
```

Comment:
This feels more like a script that users will call instead of a tutorial. I feel this should just live under `torchao/profiler/generate.py`?

Reply:
same for the README